ivy/functional/ivy/losses.py

```diff
@@ -81,24 +81,100 @@ def binary_cross_entropy(true, pred, epsilon=1e-7):
     return -(ivy.log(pred) * true + ivy.log(1 - pred) * (1 - true))
 
 
-def sparse_cross_entropy(true, pred, axis=-1, epsilon=1e-7):
+def sparse_cross_entropy(
+        true: Union[ivy.Array, ivy.NativeArray],
+        pred: Union[ivy.Array, ivy.NativeArray],
+        axis: Optional[int] = -1,
+        epsilon: Optional[float] = 1e-7,
+) -> ivy.Array:
     """Computes sparse cross entropy between logits and labels.
 
     Parameters
     ----------
     true
-        True labels as logits.
+        input array containing the true labels as logits.
     pred
-        predicted labels as logits.
+        input array containing the predicted labels as logits.
```

Comment:
It is stated that the `pred` is the `input array containing the predicted labels as logits.`. However, if you look at the docstring examples, you will find that the sum of the array values(`[0.1, 0.1, 0.7, 0.1]`) is 1. This makes the input to `pred` something like an output from the softmax layer and not the logits.

Reply:
The way predicted labels are used implies it was indeed output of softmax layer. Otherwise one needs to take softmax explicitly. Hence the correct labeling should have been ```pred```: ```input array containing predicted probabilities after taking softmax```.

Reply:
Exactly. You can make the docstring even shorter. Just write `pred`: `array containing softmax probabilities.`